VideoLocker/src/main/java/org/edx/mobile/module/registration/view/RegistrationCheckBoxView.java

```diff
@@ -82,24 +74,27 @@ public View getView() {
     }
 
     @Override
-    public void handleError(String error) {
+    public void handleError(String error, boolean requestAccessibility) {
```

Comment:
@BenjiLee We need some way for the activity to tell the control weather or not to request accessibility focus on error. If we don't then the accessibility focus will just go to every error. From a user perspective, that would mean that the last error would have the focus.

I think it's a better user experience for only the first to have the focus. So a conditional value is needed. What do you think?